network-api/networkapi/mozfest/templates/partials/homepage_banner.html

```diff
@@ -2,11 +2,13 @@
 
 <div class="homepage-banner-content container">
   <div class="row">
-    <div class="col-12 col-lg-8 dark-theme">
+    <div class="col-10 col-lg-8 dark-theme">
       <h1 class="h1-heading">{{ page.banner_heading }}</h1>
-      <p class="body-large">{{ page.banner_guide_text }}</p>
-      {% if page.banner_video_url %}
-      <a class="btn btn-primary" href="{{ page.banner_video_url }}" target="_blank" id="mozfest-home-watch-video-button">{% trans "Watch Video" %}</a>
+      {% if banner_type != "video" %}
+        <p class="body-large">{{ page.banner_guide_text }}</p>
+        {% if page.banner_video_url %}
```

Comment:
if the banner type isn't video, why would there be a banner video url?

Reply:
ahhhhhh i should rename the variable to avoid confusion